src/Api/OperationTypeDeprecationHelper.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Api;
+
+/**
+ * Class OperationTypeDeprecationHelper
+ * Before API Platform 2.1, the operation type was one of:
+ * - "collection" (true)
+ * - "item" (false).
+ *
+ * Because we introduced a third type in API Platform 2.1, we're using a string with OperationType constants:
+ * - OperationType::ITEM
+ * - OperationType::COLLECTION
+ * - OperationType::SUBCOLLECTION
+ *
+ * @internal
+ */
+final class OperationTypeDeprecationHelper
+{
+    /**
+     * @param string|bool $operationType
+     *
+     * @return string
+     */
+    public static function getOperationType($operationType): string
+    {
+        if (is_bool($operationType)) {
```

Comment:
Okay so I did try `!is_string` but I get the same error :| I'll just ignore this one for now as this is a temporary class anyway.